golem_messages/message/base.py

```diff
@@ -206,6 +212,8 @@ def deserialize_header(cls, data):
                 )
             )
 
+        from golem_messages.message import registered_message_types
+

```

Comment:
Wouldn't be better to have all imports in the top of file?

Reply:
Cyclic import (`golem_messages.message.__init__.py`). You may want to have a look at #103